src/test/testarb.annotated

```diff
@@ -13,62 +14,63 @@
         "description": "Welcome message on the Home screen",
         "placeholders": {
             "firstName": {}
-             ^^^^^^^^^[decoration]placeholder
```

Comment:
Is it intended that this is no longer decorated as a placeholder?

Reply:
I really should pay more attention to the golden test...